apps/blackout/blackout.star

```diff
@@ -0,0 +1,28 @@
+"""
+Applet: Blackout
+Summary: Blackout tidbyt
+Description: Black out TidByt during evenings (or whenever).
+Author: mabroadfo1027
+"""
+
+load("render.star", "render")
+load("schema.star", "schema")
+load("encoding/json.star", "json")
+
+def main():
+
+    pulseList=[]
+    for i in range(0, 1000):
```

Comment:
I don't think there's any need to generate a full 1000 frames for this animation. Since every 10th frame gets the fully black color, it ought to be sufficient to have 10 frames in there. The Tidbyt device will loop the animation while it's being displayed.
```suggestion
    for i in range(0, 10):
```